test/BedLikeTests.js

```diff
@@ -22,6 +22,73 @@ describe('Simple HiGlassComponent', () => {
   let hgc = null;
   let div = null;
 
+  const viewConf = {
```

Comment:
This is one thing that annoys me about linting. I had intentionally put that viewconf at the bottom because what's important is the test. Now that it's at the top, we have to scroll past it to get to the interesting test. This lengthy viewconf is now obscuring the test to conform to a style guide that is supposed to make the code more legible. Ironic.

Maybe we can disable that rule? What do you think?